src/test/store/actions.test.js

```diff
@@ -239,44 +219,6 @@ describe('selectors/getFlameGraphTiming', function() {
   });
 });
 
-describe('selectors/getCallNodeMaxDepthForStackChart', function() {
-  it('calculates the max func depth and observes of platform-detail filters', function() {
-    const store = storeWithProfile();
-    const allSamplesMaxDepth = selectedThreadSelectors.getCallNodeMaxDepthForStackChart(
-      store.getState()
-    );
-    expect(allSamplesMaxDepth).toEqual(7);
-    store.dispatch(changeHidePlatformDetails(true));
-    const jsOnlySamplesMaxDepth = selectedThreadSelectors.getCallNodeMaxDepthForStackChart(
-      store.getState()
-    );
-    expect(jsOnlySamplesMaxDepth).toEqual(5);
-  });
-
-  it('acts upon the current range', function() {
-    const store = storeWithProfile();
-    store.dispatch(addRangeFilter(0, 20));
```

Comment:
I don't see any test with `addRangeFilter` now; care to add one back, even somewhere else?

Reply:
I'd prefer to block this PR on #724 due to scope creep with this request.